### ISSUE TITLE
torch.utils.data.BufferedShuffleDataset was removed

### ISSUE THREAD

**Issue by changhiskhan** [eto-ai/rikai#599]
not documented but the class no longer exists as of torch 1.9

**Comment by eddyxu** [comment]
Hmm, so it means we lose pytorch random shuffling again?

**Comment by eddyxu** [comment]
TODO: Verify that `rikai.torch.Dataset` can work with new PyTorch's shuffling capability (or add back shuffling ourselves?)

**Comment by eddyxu** [comment]
https://pytorch.org/blog/pytorch-1.11-released/  Probably need to implement this new API
